cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/network/protocol/chunk/ChunkedPacketProvider.java

```diff
@@ -0,0 +1,49 @@
+package de.dytanic.cloudnet.driver.network.protocol.chunk;
+
+import de.dytanic.cloudnet.common.document.gson.JsonDocument;
+import de.dytanic.cloudnet.driver.network.INetworkChannel;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collection;
+
+public class ChunkedPacketProvider {
```

Comment:
Codacy found an issue: [The utility class name 'ChunkedPacketProvider' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=6005504)